pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -445,7 +449,11 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
     "LDFLAGS=-static"
   ];
 
-  preConfigure = optionalString (pythonOlder "3.12") ''
+  preConfigure = ''
+    # Attempt to purify some of the host info collection
+    sed -E -i -e 's/uname -r/echo/g' -e 's/uname -n/echo nixpkgs/g' config.guess
+    sed -E -i -e 's/uname -r/echo/g' -e 's/uname -n/echo nixpkgs/g' configure
```

Comment:
Can we use `updateAutotoolsGnuConfigScriptsHook` per the other PR? (or does it already do that in there?)

Reply:
Oh this is different